src/main/java/org/apache/pulsar/io/jcloud/BlobStoreAbstractConfig.java

```diff
@@ -89,6 +89,7 @@ public class BlobStoreAbstractConfig implements Serializable {
 
     private boolean sliceTopicPartitionPath;
 
+    private long maxBatchBytes = 10_000;
```

Comment:
please remember to verify the value in `validate()` to prevent the user provide a negative value.